lib/job/arInformationObjectXmlExportJob.class.php

```diff
@@ -38,6 +38,9 @@ public function runJob($parameters)
   {
     $this->params = $parameters;
 
+    // Do we include digital objects?
+    $this->includeDigitalObjects = array_key_exists('includeDigitalObjects', $parameters) && $parameters['includeDigitalObjects'];
```

Comment:
Cosmetic: could we move this to `arBaseJob` using `$this->params` inside `createZipForDownload`? That way it could be removed from all export jobs and not passed to that function.

Reply:
Absolutely. Fixed. Much cleaner.